packages/@sfx/sayt-driver-plugin/test/unit/common/sayt-driver-plugin.test.ts

```diff
@@ -51,6 +51,25 @@ describe('Sayt Driver Plugin', () => {
     };
   });
 
+  describe('constructor()', () => {
+    it('should save a passed product transformer as a transformProduct property', () => {
+      const productTransformer = ((product) => 123) as any;
```

Comment:
Prefer to annotate types instead of casting:

```suggestion
      const productTransformer: any = (product) => 123;
```